l/librdkafka/librdkafka_ubi_9.3.sh

```diff
@@ -41,15 +41,22 @@ if ! ./configure --install-deps ; then
     exit 1
 fi
 
+if ! make; then
+    echo "------------------$PACKAGE_NAME: build fails---------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Build_Fails"
+    exit 1
+fi
+
 if ! make install; then
-    echo "------------------$PACKAGE_NAME:install_fails---------------------"
+    echo "------------------$PACKAGE_NAME: install_fails---------------------"
     echo "$PACKAGE_URL $PACKAGE_NAME"
     echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Install_Fails"
     exit 1
 fi
 
 if ! make -C tests run_local_quick; then
-    echo "------------------$PACKAGE_NAME:install_fails---------------------"
+    echo "------------------$PACKAGE_NAME: install_success---------------------"
```

Comment:
can be changed to install_success_but_test_fails

Reply:
@prachikurade1 done with the required changes.